tekton-task-images/helm/Dockerfile

```
@@ -2,14 +2,25 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
 
 USER root
 
+ENV YQ_VERSION=4.11.2
```

Unless the `yq` version is needed as an ENV var in the runtime....
```suggestion
ARG YQ_VERSION=4.11.2
```